### PR TITLE
Security guide writer's review

### PR DESCRIPTION
**Pull request by MichalMaler**
The guides I touched in this PR are well-written overall.
I am just applying one sentence per line, removing unnecessary white space, and fixing a style/grammar a little bit here and there.
Taking a PTO for the rest of the year :)
Looking forward to any feedback that will help to ameliorate my PR and enhance these good looking docs!

### CODE REVIEW

**Comment by sberyozkin** [comment]
>Taking a PTO for the rest of the year :)

Enjoy it Michal @MichalMaler

File: docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc [hunk 1]
```diff
@@ -368,7 +384,7 @@ s| @DenyAll | Specifies that no security roles are allowed to invoke the specifi
 s| @PermitAll | Specifies that all security roles are allowed to invoke the specified methods.
 
 `@PermitAll` lets everybody in, even without authentication.
-s| @RolesAllowed | Specifies the list of security roles permitted to access methods in an application.
+s| @RolesAllowed | Specifies the list of security roles authorized to access methods in an application.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc:387]:
```suggestion
s| @RolesAllowed | Specifies the list of security roles allowed to access methods in an application.
```

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc:387]: `authorized` may be read by some users as if the authorization decision has been made, so may be `allowed` would be better, `allowed` is already used just above as well

File: docs/src/main/asciidoc/security-basic-authentication.adoc [hunk 2]
```diff
@@ -11,22 +11,20 @@ include::_attributes.adoc[]
 :topics: security,authentication,basic-authentication,http
 :extensions: io.quarkus:quarkus-vertx-http,io.quarkus:quarkus-elytron-security-jdbc,io.quarkus:quarkus-elytron-security-ldap,io.quarkus:quarkus-security-jpa-reactive
 
-HTTP Basic authentication is one of the least resource-demanding techniques that enforce access controls to web resources.
-You can secure your Quarkus application endpoints with HTTP Basic authentication.
-Quarkus provides a built-in authentication mechanism for Basic authentication.
+HTTP Basic authentication is a low-resource method for enforcing access controls to web resources.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-basic-authentication.adoc:14]: Michal, that sentence which as far as I recall you created awhile back is clearer, `low-resource` reads unusual, and may be confusing

File: docs/src/main/asciidoc/security-basic-authentication.adoc [hunk 3]
```diff
@@ -11,22 +11,20 @@ include::_attributes.adoc[]
...
+You can secure your Quarkus application endpoints by using HTTP Basic authentication.
+Quarkus includes a built-in authentication mechanism for Basic authentication.
 
-Basic authentication uses fields in the HTTP header and does not require HTTP cookies, session identifiers, or login pages.
+Basic authentication operates through fields in the HTTP header and does not rely on HTTP cookies, session identifiers, or login pages.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-basic-authentication.adoc:18]:
```suggestion
Basic authentication uses fields in the HTTP header and does not rely on HTTP cookies, session identifiers, or login pages.
```

`Uses` reads so much simpler, hope you agree :-). The simpler, the better IMHO

**Review by sberyozkin — COMMENTED** [review]
Thanks for these improvements, @MichalMaler. I left a few minor suggestions

File: docs/src/main/asciidoc/security-basic-authentication.adoc (hunk 2 above)
Comment (MichalMaler) [reply docs/src/main/asciidoc/security-basic-authentication.adoc:14]: OK, will keep it as originally written :)

**Comment by MichalMaler** [comment]
OK folks. I applied @sberyozkin Review. It is ready for your eyes now.
All the best to 2024 and hope you recharge well and will experience some magical times with your loved ones.
Cheers!

**Comment by sberyozkin** [comment]
Thanks @MichalMaler, same wishes to you

**Review by sberyozkin — APPROVED** [review]

**Review by rolfedh — COMMENTED** [review]
LGTM. Excellent edits.

**Pull request merged**